src/components/WebexData/WebexData.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const AdapterContext = React.createContext();
+
+export default function WebexData(props) {
```

Comment:
I think the team was leaning towards renaming this `WebexDataProvider`

Reply:
yeah